src/hyperv/hyperv_network_api_v2.c

```diff
@@ -71,103 +71,141 @@ hyperv2MsvmVirtualSwitchToNetwork(virConnectPtr conn,
     return result;
 }
 
-/* exported API functions */
-
-int
-hyperv2ConnectListNetworks(virConnectPtr conn, char **const names, int maxnames)
+static int
+hyperv2GetVirtualSwitchList(hypervPrivate *priv,
+                            const char *filter,
```

Comment:
again, this is stack-allocated by the caller so once it gets out of scope it will cleanup